src/test/ui/rfc-2093-infer-outlives/enum.rs

```diff
@@ -19,15 +21,17 @@ enum Foo<'a, T> {
 }
 
 // Type U needs to outlive lifetime 'b
-struct Bar<'b, U> {
-    field2: &'b U //~ ERROR the parameter type `U` may not live long enough [E0309]
+#[rustc_outlives]
+struct Bar<'b, U> { //~ ERROR 25:1: 27:2: rustc_outlives
+    field2: &'b U
 }
 
 
 
 // Type K needs to outlive lifetime 'c.
-enum Ying<'c, K> {
-    One(&'c Yang<K>) //~ ERROR the parameter type `K` may not live long enough [E0309]
```

Comment:
huh this test looks like it was kind of broken before, eh?

Reply:
I dont think it was broken. The `rustc_outlives` attr just gives more succinct errors. Also because we are stabilizing we need the attr to evoke the outlives error.